tests/suites/test_suite_ssl.function

```diff
@@ -36,10 +36,17 @@ void log_analyzer( void *ctx, int level,
     }
 }
 
+/* Defaults for min/max version tests (used for arguments as well) */
```

Comment:
Just "Default" not "Defaults" now probably. Otherwise it would probably be more accurate to define it as the minor version invalid value. Then this value is used as a default value for some fields.